pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -587,6 +587,7 @@ class GNIRSIFUSpectrograph(GeminiGNIRSSpectrograph):
     #   * Need to store a wavelength solution for different grating options (Note, the Holy Grail algorithm works pretty well, most of the time)
     name = 'gemini_gnirs_ifu'
     pypeline = 'SlicerIFU'
+    allowed_extensions = ['.fits', '.fits.bz2']
```

Comment:
What happens is the user has a set of fits files in their `Raw/` directory, and also the exact same files but compressed?

Reply:
This is consistent with the previous behavior: the code would find both.  At least now, users can limit the search to one or the other using the `--extension` command-line argument.  I added some text in the setup doc about this.